lib/aptible/cli/subcommands/metric_drain.rb

```diff
@@ -0,0 +1,119 @@
+module Aptible
+  module CLI
+    module Subcommands
+      module MetricDrain
+        def self.included(thor)
+          thor.class_eval do
+            include Helpers::Token
+            include Helpers::Database
+            include Helpers::MetricDrain
+
+            desc 'metric_drain:list', 'List all Metric Drains'
+            option :environment
+            define_method 'metric_drain:list' do
+              Formatter.render(Renderer.current) do |root|
+                root.keyed_list('description') do |node|
+                  scoped_environments(options).each do |account|
+                    account.metric_drains.each do |drain|
+                      node.object do |n|
+                        ResourceFormatter.inject_metric_drain(n, drain, account)
+                      end
+                    end
+                  end
+                end
+              end
+            end
+
+            desc 'metric_drain:create:influxdb HANDLE '\
+                 '--db DATABASE_HANDLE --environment ENVIRONMENT',
+                 'Create an InfluxDB Metric Drain'
+            option :db, type: :string
+            option :environment
+
+            define_method 'metric_drain:create:influxdb' do |handle|
+              account = ensure_environment(options)
+              database = ensure_database(options)
```

Comment:
This is implicitly enforcing that the destination database must be in the same account that you're creating the drain for. Should we include that in the description, or would a note in the documentation about the limitation suffice?